### PR TITLE
SA-2 :Query Optimization

### PR DESCRIPTION
**Pull request by theparasgami**
Closes: #1133 

## Proposed changes
This PR includes changing of some old queries which are inefficient and taking longer time to a new version.
### Brief description of what is fixed or changed
The old queries were replaced by new django-orm queries which are fast and optimized version of previous one.
## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by theparasgami** [comment]
@PranshuNayak 
@samay-rgb

File: FusionIIIT/applications/central_mess/handlers.py [hunk 1]
```diff
@@ -610,7 +610,7 @@ def generate_bill():
                                        month=previous_month,
                                        year = previous_month_year,
                                        total_bill=total):
-            print("exists")
+           1
```
Comment (samay-rgb) [inline FusionIIIT/applications/central_mess/handlers.py:606]: Is this condition required? [exists()](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#django.db.models.query.QuerySet.exists) can be used to check if an object exists.

File: FusionIIIT/applications/central_mess/handlers.py [hunk 2]
```diff
@@ -541,14 +541,14 @@ def add_mess_committee(request, roll_number):
     print("\n\n\\n\n\n\n\n")
     print(roll_number)
     mess = Messinfo.objects.get(student_id__id=roll_number)
-    print(mess)
+    print(mess)
```
Comment (samay-rgb) [inline FusionIIIT/applications/central_mess/handlers.py]: Please remove unnecessary print statements.

Comment (theparasgami) [reply FusionIIIT/applications/central_mess/handlers.py]: Removed In a new Commit

File: FusionIIIT/applications/central_mess/handlers.py (hunk 1 above)
Comment (theparasgami) [reply FusionIIIT/applications/central_mess/handlers.py:606]: Although it won't matter to pass the condition as we have to check if the object is already exist and then do nothing, used the exists() function in the if condition now in the latest commit.

Comment (samay-rgb) [reply FusionIIIT/applications/central_mess/handlers.py:606]: You can remove that if condition since its not used to make any changes.

**Review by samay-rgb — APPROVED** [review]

**Review by PranshuNayak — APPROVED** [review]
LGTM!

**Comment by PranshuNayak** [comment]
@akshatnema

**Pull request merged**